food_trucks/demo/pos_session.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+  <record id="pos_session_18" model="pos.session">
+    <field name="config_id" ref="pos_config_2"/>
+    <field name="name">POS/00018</field>
+    <field name="start_at" eval="(datetime.today() + timedelta(days=1)).strftime('%Y-%m-%d 13:53:27')"/>
+    <field name="state">opened</field>
+  </record>
+  <record id="pos_session_15" model="pos.session">
+    <field name="config_id" ref="pos_config_2"/>
+    <field name="name">POS/00015</field>
+    <field name="user_id" ref="base.user_admin"/>
+    <field name="start_at" eval="(datetime.today() + timedelta(days=1)).strftime('%Y-%m-%d 14:19:34')"/>
+    <field name="stop_at" eval="(datetime.today() + timedelta(days=1)).strftime('%Y-%m-%d 14:20:36')"/>
+    <field name="state">closed</field>
+  </record>
```

Comment:
I don't think we need the whole historic here. You could assign everything to current pos.session pos_session_18 and remove all others